handlers/dimension/utils.go

```diff
@@ -0,0 +1,60 @@
+package dimension
+
+import (
+	"errors"
+	"github.com/ONSdigital/dp-dd-api-stub/stub"
+	"github.com/ONSdigital/dp-dd-api-stub/models"
+	"fmt"
+	"os"
+	"encoding/json"
+)
+
+func findDimension(datasetID string, dimensionID string)  (dimension *models.Dimension, err error) {
+	raw, err := stub.Asset("data/datasets/" + datasetID + "/dimensions.json")
+	if err != nil {
+		fmt.Println(err.Error())
+		os.Exit(1)
+	}
+
+	var dimensions []models.Dimension = []models.Dimension{}
+	json.Unmarshal(raw, &dimensions)
+
+	for _, dimension := range dimensions {
+		if dimension.ID == dimensionID {
+			return &dimension, nil
+		}
+	}
+	return &models.Dimension{}, errors.New("Dimension id " + dimensionID + " not found.")
```

Comment:
It's not dropping information, the caller knows exactly what dimension ID it passed in and can choose to log it along with the error if it's useful